scrapers/GASM.yml

```diff
@@ -0,0 +1,22 @@
+name: GASM
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - gasm.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="gqText"]/span[@class="gqTitle" and 2]
+      Performers:
+        Name: //div[@class="gqOwner"]/a
+      Tags:
+        Name: //div[@class="content tag"]/a
+      Details:
+        selector: //span[@class="gqDescription"]
```

Comment:
In scenes where the scene is part of a movie we get a Part of .... at the start of the description, eg `https://www.gasm.com/post/details/20936` `https://www.gasm.com/post/details/21234`
Using
```yaml
      Details:
        selector: //span[@class="gqDescription"]/text()
        concat: "\n\n"
        postProcess:
          - replace:
              - regex: '^Part\sof\s*\n+'
                with:
      Movies:
        Name: //span[@class="gqDescription"]/a[@class="gqTitle"]
```
looks better imo.

Reply:
Good point on the movie have added it in :-)